packages/ra-supabase-core/package.json

```diff
@@ -18,19 +18,21 @@
         "ra-core": "^5.0.1"
```

Comment:
don't you need to update this line too?

Reply:
This isn't actually required but it doesn't hurt to do it